src/client/src/rt-components/tear-off/ExternalWindow.tsx

```diff
@@ -55,7 +55,8 @@ const ExternalWindow: FC<ExternalWindowProps> = ({
         externalWindow.close()
       }
     }
-  }, [config, onUnload, platform.window])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
any change to `config, onUnload, platform.window` was executing script which was closing the window

Reply:
I don't like this fix since we should chose a different strategy rather than disabling TS rules. But this is how it was before, let's leave it until we come up with a better approach

Reply:
I agree that we should find a way of removing the tslint exception. It seems bad that calling `externalWindow.close()` here can close a different window.